CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+## Changed
+
+- chore: upgrade Fluent Bit to v1.6.10-sumo-3 [#2730]
```

Comment:
Please add information about changing bitnami repo here too

Reply:
I put the repo change under the fixed section...since it seemed like a regression bug.